mentors/python/d.json

```
@@ -13,6 +13,13 @@
     "link_text": "Website",
     "bio": "Love python, easy to read and fun to write"
   },
+  {
+    "github_username": "dbarnett",
+    "name": "David Barnett",
+    "link_url": null,
+    "link_text": null,
+    "bio": "I love python because it's easy to collaborate in. I can read almost anyone's python!"
+  }
```

```suggestion
  },
```
Need a comma here....